---a/net/minecraft/entity/IRangedAttackMob.java
+++b/net/minecraft/entity/IRangedAttackMob.java
@@ -1,7 +1,7 @@
 package net.minecraft.entity;
 
-public interface IRangedAttackMob
-{
+public interface IRangedAttackMob {
+
     void attackEntityWithRangedAttack(EntityLivingBase p_82196_1_, float p_82196_2_);
-}
+}
